src/assets/data/edition/series/2/section/2a/m22/source-description.json

```diff
@@ -1745,7 +1745,10 @@
                     },
                     {
                         "item": "M 23 Sk1",
-                        "itemLinkTo": {},
+                        "itemLinkTo": {
+                            "complexId": "m23",
+                            "sheetId": "M_23_Sk1a"
+                        },
                         "itemDescription": "(Skizze zu Studienkomposition für Klavier M 23: Thema)",
```

Comment:
```suggestion
                        "itemDescription": "(Skizze zu Studienkomposition für Klavier M 23: Textfassung 1 Thema)",
```

Reply:
@chael-mi Was machen wir mit den noch offenen Vorschlägen in diesem PR?

Reply:
@chael-mi Die Vorschläge hier sind noch nicht inkludiert. Also grundsätzlich mit Angabe der Textfassung, oder ohne?